CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Added flake8 check workflow on pull_request event [#321](https://github.com/scanapi/scanapi/pull/321)
```

Comment:
Hey @camilamaia 

This change already merged and release in 2.2.0 version, but wasn't mentioned in changelog file. I mentioned this as unreleased or mentioned this change in 2.2.0 release? :confused: 

P.S. I'm so bad to explain something :disappointed:

Reply:
@hebertjulio it is all good! The explanation was perfect 👌 

I removed manually this change because [it does not affect directly the final user](https://github.com/scanapi/scanapi/wiki/Changelog#what-warrants-a-changelog-entry). So you can remove it too, no need to add a new changelog entry for this.